modules/nodes-namespaced-nodelevel-sysctls.adoc

```diff
@@ -21,7 +21,7 @@ Additionally, most of the sysctls in the `net.*` group are known to be namespace
 Sysctls that are not namespaced are called _node-level_ and must be set
 manually by the cluster administrator, either by means of the underlying Linux
 distribution of the nodes, such as by modifying the `_/etc/sysctls.conf_` file,
-or by using a daemon set with privileged containers. You can use the Node Tuning Operator to set _node-level_ sysctls.
+or by using a daemon set with privileged containers. You can use the xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[Node Tuning Operator] to set _node-level_ sysctls.
```

Comment:
```suggestion
or by using a daemon set with privileged containers. You can use the Node Tuning Operator to set _node-level_ sysctls. 
[NOTE]
====
This information can be found in the "Scalability and performance" section of the {product-title} documentation. Look at the sub-section titled "Using the Node Tuning Operator".
==== 
```

Reply:
@anandrece I have provided peer-review feedback that should solve the problem of using the xref.  This should also allow you to produce a correct preview, needed for merge review.